<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegionSelectionWidget</class>
 <widget class="QWidget" name="RegionSelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegionSelectionWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Zoom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="_zoomSpinBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="_timeStatusLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>3</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>3</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QScrollArea" name="_scrollArea">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>298</width>
         <height>298</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="_startLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="_startImageLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="_endLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>End</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="_endImageLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>Image</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_regionsGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>View Regions</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="_newButton">
          <property name="text">
           <string>New</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">_regionsButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="_allButton">
          <property name="text">
           <string>All</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">_regionsButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="_allNoTimeButton">
          <property name="text">
           <string>All (no time)</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">_regionsButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="_selectedButton">
          <property name="text">
           <string>Selected</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">_regionsButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="_regionComboBox"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="VideoControlWidget" name="_videoControls" native="true"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_selectButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_cancelButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoControlWidget</class>
   <extends>QWidget</extends>
   <header>videocontrolwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_zoomSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>RegionSelectionWidget</receiver>
   <slot>zoom_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_selectButton</sender>
   <signal>clicked()</signal>
   <receiver>RegionSelectionWidget</receiver>
   <slot>select_region()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>770</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_regionsButtonGroup</sender>
   <signal>buttonToggled(int,bool)</signal>
   <receiver>RegionSelectionWidget</receiver>
   <slot>region_display_mode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_regionComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>RegionSelectionWidget</receiver>
   <slot>region_combobox_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>RegionSelectionWidget</receiver>
   <slot>reset_enter_region()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>891</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="_regionsButtonGroup"/>
 </buttongroups>
</ui>
